packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -395,6 +396,17 @@ export default {
             return invalidFieldCount === 1 ?
                 this.$t('errorMessages.singleFieldError') :
                 this.$t('errorMessages.multipleFieldErrors', { errorCount: invalidFieldCount });
+        },
+
+        formattedMobileNumberForScreenReader () {
+            const usesAreaCode = this.customer.mobileNumber.startsWith('+');
+            const pauseIndices = usesAreaCode ? [6, 9] : [4, 7]; // TODO: Split the number at different indices depending on country
+            return Array.from(this.customer.mobileNumber, (digit, index) => {
+                if (pauseIndices.includes(index)) {
+                    return `${digit}.`;
+                }
+                return digit;
+            }).join(' ');
```

Comment:
I feel like this is a bit mental and could maybe just pause after every third digit, but felt like I should structure it how we tend to in the UK 🙃

Reply:
I personally think we should simplify this and do what you suggested. A pause after 3 digits or so, or even just read a digit at a time and see how fast or how slow that's read by the screenreader. 

We need to support several countries with different phone numbers lengths and ways to structure them.
Even within one country, people can structure it or read it differently.

I might be completely wrong here, but I'd prefer the number to be read at a decent speed, one at a time, rather than the website trying to be too smart and make things more confusing.

cc @ross-gratton.

Reply:
I am more than happy to do that. Less code is always better. I think having no periods at all works but I was just following a suggestion given in the ticket.

Reply:
We have decided to remove fulls tops all together as the issue was the number being read out as a whole number and now it isn't. And as you mentioned, this will need to cover other territories so it's safer this way.